src/lib/libast/misc/spawnvex.c

```diff
@@ -704,20 +704,26 @@ pid_t spawnvex(const char *path, char *const argv[], char *const envv[], Spawnve
         int m;
         Spawnvex_noexec_t nx;
         int msg[2];
+#if _lib_vfork
```

Comment:
Legacy code keeps these code blocks under ```_real_vfork``` macro. If there are any regressions around this code, it will make debugging easier if we rename this to ```_real_vfork```.